xrpl/asyncio/clients/websocket_base.py

```diff
@@ -7,7 +7,7 @@
 from typing import TYPE_CHECKING, Any, Dict, Optional, cast
 
 from typing_extensions import Final
-from websockets.legacy.client import WebSocketClientProtocol, connect
+from websockets import client as websocket_client
```

Comment:
Why was this changed in this PR? (Maybe it could be split out?)

Reply:
bumped the websocket library while here :)